packs/equipment/camouflaging-chromatophores-greater.json

```diff
@@ -1,6 +1,6 @@
 {
     "_id": "v2aOpKhNP0uIZFER",
-    "img": "icons/creatures/reptiles/chameleon-camouflage-green-brown.webp",
+    "img": "systems/pf2e/icons/default-icons/equipment.svg",
```

Comment:
Was this intentional?

Reply:
No, this was not - this undid the previous round - I'll redo the request in a bit.